SimPPS/RPDigiProducer/src/DeadChannelsManager.cc

```diff
@@ -0,0 +1,58 @@
+#include "SimPPS/RPDigiProducer/interface/DeadChannelsManager.h"
+#include "CondFormats/CTPPSReadoutObjects/interface/TotemSymbId.h"
+#include "CondFormats/CTPPSReadoutObjects/interface/TotemDAQMapping.h"
+#include "DataFormats/CTPPSDetId/interface/TotemRPDetId.h"
+#include "SimPPS/RPDigiProducer/interface/RPDisplacementGenerator.h"
+#include <map>
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+DeadChannelsManager::DeadChannelsManager() {
+	analysisMaskPresent = false;
+}
+
+DeadChannelsManager::DeadChannelsManager(edm::ESHandle<TotemAnalysisMask> _analysisMask) {
+	analysisMask = _analysisMask;
```

Comment:
There seems to be no specific need of `ESHandle`, if that is really the case, it would be better to pass `TotemAnalysisMask` as `const` pointer. Then the `analysisMaskPresent` boolean would not be needed either (well, it is not really needed with `ESHandle` either).